tests/TryCatch/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -120,9 +120,7 @@ contract FeedConsumer {
         // Permanently disable the mechanism if there are
         // more than 10 errors.
         require(errorCount < 10);
-        try
-            feed.getData(token)
-        returns (
+        try feed.getData(token) returns (
```

Comment:
is this ok?

Reply:
I like it this way better but could be convinced otherwise